argus/client/windows.py

```diff
@@ -147,12 +147,12 @@ def copy_file(self, filepath, remote_destination):
         commands = []
         for command in _base64_read_file(filepath):
             remote_command = (
-                "{content} >> '{remote_destination}'"
+                r'powershell "{content}" >> "{remote_destination}"'
                 .format(content=get_string_cmd.format(command),
                         remote_destination=remote_destination))
 
             commands.append(remote_command)
-        self._run_commands(commands, commands_type=util.POWERSHELL)
+        self._run_commands(commands, commands_type=util.CMD)
```

Comment:
We will get this error `The command line is too long` because the content is going to be encoded by the read method then encoded again because the command is Powershell. We can keep the command as Powershell type but we need to read 2048 bites not 8192 like we do now, I don't thinks it is worth it .